src/containers/LoadingScreen/components/CustomerSlider/CustomerSlider.js

```diff
@@ -6,14 +6,38 @@ import { Customers } from './Customers'
 import './CustomSlider.css'
 
 export default class CustomerSlider extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      randomCustomers: []
+    }
+  }
+
   componentDidMount(): void {
     this.interval = setInterval(() => this.slider.slickNext(), 5000)
+    this.state.randomCustomers = this.randomCustomers();
   }
 
   componentWillUnmount() {
     clearInterval(this.interval)
   }
 
+  randomCustomers() {
+    const min = 0;
+    const max = Customers.length - 1
+    var randomCustomers = []
+    var counter = 0;
+    while (randomCustomers.length != 6) {
+      var randomNumber = Math.floor(Math.random() * (max - min + 1) + min)
+      if (!randomCustomers.includes(Customers[randomNumber])) {
```

Comment:
Ich bin mir nicht sicher, ob includes bei object-vergleichen in einem Array immer so zuverlässig ist. Mach das mal lieber mit find und dann matched du das anhand de classNames. Also sowas wie `!randomCustomers.find(customer => customer.className !== Customers[randomNumber].className)`